ui/src/app/routing/signed-in-app-routing.tsx

```diff
@@ -102,17 +110,24 @@ export const SignedInRoutes = () => {
     <AppRoute exact path='/admin/workspaces/:workspaceNamespace/:nbName'>
       <AdminNotebookViewPage routeData={{pathElementForTitle: 'nbName', minimizeChrome: true}}/>
     </AppRoute>
+    <AppRoute exact path='/data-access-requirements'>
+      <DataAccessRequirementsPage routeData={{title: 'Data Access Requirements'}}/>
+    </AppRoute>
     <AppRoute exact path='/data-code-of-conduct'>
       <DataUserCodeOfConductPage routeData={{title: 'Data User Code of Conduct', minimizeChrome: true}}/>
     </AppRoute>
     <AppRoute exact path='/profile'>
       <ProfilePage routeData={{title: 'Profile'}}/>
     </AppRoute>
-    <AppRoute exact path='/nih-callback'>
-      <HomepagePage routeData={{title: 'Homepage'}}/>
+    <AppRoute exact path={NIH_CALLBACK_PATH}>
```

Comment:
(comment also applies to RAS)

Both pages process the callback requests in the same way, by updating the user via the API.  The difference is the UX post callback.

FF = false (current behavior) -> unregistered users are redirected to the homepage which **includes** the Registration Dashboard

FF = true -> unregistered users are redirected to Data Access Requirements which is **separate** from the homepage